.travis.yml

```diff
@@ -17,12 +17,10 @@ jdk:
   - oraclejdk8
 
 install:
-  - mvn clean install -DskipTests -Dassembly.skipAssembly=true > /dev/null
+  - bash .travis_install.sh
 
 script:
-  - cd $PINOT_MODULE
-  - cp ../.travis_cmd.sh .
-  - bash .travis_cmd.sh
+  - bash .travis_test.sh
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
```

Comment:
I guess it would make more sense if it was internal github.

On Thu, Jun 2, 2016 at 11:20 AM, Adwait Tumbde <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/linkedin/pinot/pull/140#discussion_r65591736>:
>
> >
> >  script:
> > -  - cd $PINOT_MODULE
> > -  - cp ../.travis_cmd.sh .
> > -  - bash .travis_cmd.sh
> > +  - bash .travis_test.sh
> >
> >  after_success:
> >    - bash <(curl -s https://codecov.io/bash)
>
> There is no real difference in pulling the script vs storing it on github.
> Both can be compromised.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/linkedin/pinot/pull/140/files/0b806a490eba7a6250c08dab0ce795029c2f111e#r65591736>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ADSJ7mSr2tJN66ER667g7MJ-3kxHxjXbks5qHx8LgaJpZM4IsJ6Y>
> .
>